plugins/LadspaEffect/caps/dsp/Sine.h

```diff
@@ -25,6 +25,7 @@
 	02111-1307, USA or point your web browser to http://www.gnu.org.
 */
 
+
```

Comment:
No need to change this file any more.

Reply:
That was a remnant of adding basics.h everywhere. Will fix it soon ish, when 9 get back to my pc.

Reply:
fixed